showcase-docs/getting-started.md

```diff
@@ -6,7 +6,7 @@ There are several different methods for running the Backstage Showcase app today
 
 The easiest and fastest method for getting started: Backstage Showcase app, running it locally only requires a few simple steps.
 
-1. Copy `app-config.basic.yaml` and rename it as `app-config.local.yaml`.
+1. Copy `app-config.example.yaml` and rename it as `app-config.local.yaml`.
```

Comment:
why not using `app-config.local.yaml` and avoid the rename step?

Reply:
I think `example` is fine...  `local` is by default `.gitignored` so users can change it to their liking if they are developing locally... This way we separate the example deployment (the source for example config) and the autoloaded local defaults. I would keep it as is